### PR TITLE
chore: add sysdig links/monitoring to shipit

### PR DESCRIPTION
**Pull request by dleard**
I think the iframe will have an issue unless we make the dashboard public in sysdig?

### CODE REVIEW

**Comment by matthieu-foucault** [comment]
Let's merge and see what it looks like :smile:

**Comment by matthieu-foucault** [comment]
image

**Comment by matthieu-foucault** [comment]
See https://cas-shipit.apps.silver.devops.gov.bc.ca/bcgov/cas-ciip-portal/dontdeployme

**Comment by matthieu-foucault** [comment]
The links don't appear to be showing. I was already logged in sysdig, so the monitoring shows up. We just need to figure out how to make it bigger

**Comment by matthieu-foucault** [comment]
Nevermind, the links appeared after shipit synchronized. Looks like the PVC usage dashboard wasn't share with all teams though

**Comment by matthieu-foucault** [comment]
See https://github.com/Shopify/shipit-engine/blob/236ffbfefb55ffc5faaa2ec27f72bb173106b3d4/examples/shipit.yml#L17 to set the iframe width and height

**Comment by matthieu-foucault** [comment]
I don't think that we need the monitoring to show up in anything else than prod. I'm not sure how the environment-specific yml files are handled though. Maybe try adding a `shipit.dontdeployme.yml`, which only has the monitoring settings, to see if it merges it with the base `shipit.yml`

**Comment by dleard** [comment]
Cool thanks, I'll play with the iframe & PVC dashboard. Did you create a new shipit stack and just point it to this branch to make the 'dontdeployme'?

**Comment by matthieu-foucault** [comment]
Yes I created a shipit stack, that was the easiest to test this.

**Comment by dleard** [comment]
shipit.dontdeployme,yml overrides the default shipit.yml, it does not merge with it.
The iframe has been enlarged but it has some permission / data issues. I set the dashboard to be shared with all teams but if you are not logged into the CIIP namespace's sysdig team the data will show up as `No data` because the other namespaces can't find those pods

**Comment by dleard** [comment]
@matthieu-foucault Per your question in stand-up today: Yes, you can log in to sysdig from the iframe if you're not already logged in

File: shipit.dontdeployme.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+dependencies:
+  override: []
+deploy:
+  override:
+    - echo 'Not to be deployed'
```
Comment (matthieu-foucault) [inline shipit.dontdeployme.yml:5]: Looks good. Let's just copy what's in the default shipit.yml and rename this to `shipit.prod.yml`

File: shipit.dontdeployme.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+review:
+  checklist: []
```
Comment (matthieu-foucault) [inline shipit.dontdeployme.yml:7]:
```suggestion
  checklist:
    - Is today not Friday?
```

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

File: shipit.prod.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        timeout: 2460
+review:
+  checklist:
+    - Is today not Friday?
+  monitoring:
+    - iframe: https://app.sysdigcloud.com/#/dashboards/222088?last=3600
+      width: 1200
+      height: 800
+ci:
+  hide:
```
Comment (matthieu-foucault) [inline shipit.prod.yml:16]:
```suggestion
  require:
```
the release approval is actually required for prod (and test)

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**